app/helpers/pulfalight_helper.rb

```diff
@@ -28,7 +28,11 @@ def repository_thumbnail
                 repository_thumbnail_path
               end
 
-    image_tag(img_src, alt: "", class: "img-fluid float-left")
+    image_tag(img_src, alt: "Repository", class: "img-fluid float-left")
```

Comment:
I wonder if we could use the document to put the name of the repository in the alt text